.github/gh_matrix_builder.py

```diff
@@ -158,29 +166,35 @@ def macos_config(overrides):
             "pg": PG13_LATEST,
             "cc": "clang-14",
             "cxx": "clang++-14",
-            "installcheck_args": pg13_installcheck_args,
+            "installcheck_args": pg13_installcheck_args_pr,
         }
     )
 )
 
 m["include"].append(
-    build_debug_config({"pg": PG14_LATEST, "installcheck_args": pg14_installcheck_args})
+    build_debug_config(
+        {"pg": PG14_LATEST, "installcheck_args": pg14_installcheck_args_pr}
+    )
 )
 
 m["include"].append(
-    build_debug_config({"pg": PG15_LATEST, "installcheck_args": pg15_installcheck_args})
+    build_debug_config(
+        {"pg": PG15_LATEST, "installcheck_args": pg15_installcheck_args_pr}
+    )
 )
 
 # test latest postgres release in MacOS
 m["include"].append(
     build_release_config(
-        macos_config({"pg": PG15_LATEST, "installcheck_args": pg15_installcheck_args})
+        macos_config(
+            {"pg": PG15_LATEST, "installcheck_args": pg15_installcheck_args_pr}
+        )
     )
 )
 # test latest postgres release without telemetry
 m["include"].append(
     build_without_telemetry(
-        {"pg": PG15_LATEST, "installcheck_args": pg15_installcheck_args}
+        {"pg": PG15_LATEST, "installcheck_args": pg15_installcheck_args_pr}
```

Comment:
there is yet another set of IGNORES around line #212

Reply:
Thats for non-PR CI runs (scheduled etc.) We want to run these tests for scheduled runs.